spec/controllers/posts_controller_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe PostsController, type: :controller do
```

Comment:
- [ ] The type: should be :response instead of :controller, the type: :controller specifies that the tests are focused on testing the UsersController and its actions. This type indicates that the tests are controller tests and not request tests. To write request tests, you would need to specify type: :request instead of type: :controller. Request tests are used to test the entire HTTP request-response cycle and are typically used to test endpoints that respond to specific HTTP verbs (such as GET, POST, PUT, DELETE, etc.).
- For this millstone, it is expected to write tests for get requests therefore you should use the :request type